main.js

```diff
@@ -1,4 +1,137 @@
 /* eslint-disable */
+const filterBtn = document.getElementById('filterBtn-js');
+const filter = document.getElementById('filter-js');
+
+filterBtn.addEventListener('click', () => {
+  filter.style.display = 'block';
+})
+
+const adultCount = document.getElementById('adultCount');
+const childrenCount =document.getElementById('childrenCount');
+const roomCount = document.getElementById('roomCount');
+const minusBtn = document.querySelectorAll('.minus-filter-btn');
+const plusBtn = document.querySelectorAll('.plus-filter-btn');
+const childAgeBlock = document.querySelectorAll('.filter-childAgeBlock');
+
+//Adults
+
+minusBtn[0].addEventListener('click', () => {
+  if (adultCount.textContent !== `${0}`) {
```

Comment:
I have some questions :)
1. Why do you use `${0}` instead of "0"?
2. Why do you use 0, 10, 30 and etc? I don't know what is the numbers. Can you create constants for this numbers? So, it helps other developers understand what is the numbers